src/components/Button.tsx

```diff
@@ -1,10 +1,13 @@
 interface ButtonProps {
```

Comment:
```suggestion
import { ButtonHTMLAttributes } from 'react'

interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
```

Этим мы показываем, что расширяем пропсы стандартной html-кнопки. Теперь функцию можно вешать просто через onClick, т.к. этот пропс есть у стандартной кнопки